Libraries/Opc.Ua.Configuration/IApplicationConfigurationBuilder.cs

```diff
@@ -492,6 +538,13 @@ public interface IApplicationConfigurationBuilderSecurityOptions :
         /// <param name="keySize">The minimum RSA key size to accept.</param>
         IApplicationConfigurationBuilderSecurityOptions SetMinimumCertificateKeySize(ushort keySize);
 
+        /// <summary>
+        /// The minimum ECDSA key size to accept.
+        /// By default the key size is set to <see cref="CertificateFactory.DefaultKeySize"/>.
+        /// </summary>
+        /// <param name="keySize">The minimum ECDSA key size to accept.</param>
+        IApplicationConfigurationBuilderSecurityOptions SetMinimumECCertificateKeySize(ushort keySize);
```

Comment:
for ECC the setting doesn't make sense

Reply:
Why not ? Aren't there multiple ECC certificate key sizes to be accepted (256, 384 or 521) ?